README.md

```diff
@@ -1 +1,68 @@
 # RFM9x
+
```

Comment:
Please add usage samples for all the methods (see https://github.com/electricimp/OAuth/tree/develop for example)